packages/core/strapi/lib/commands/utils/logger.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const chalk = require('chalk');
+
+/**
+ * @typedef {{ silent?: boolean; debug?: boolean; timestamp?: boolean; }} LoggerOptions
+ */
+
+/**
+ * @typedef {object} Logger
+ * @property {number} warnings
+ * @property {number} errors
+ * @property {(...args: any[]) => void} debug
+ * @property {(...args: any[]) => void} info
+ * @property {(...args: any[]) => void} warn
+ * @property {(...args: any[]) => void} error
+ * @property {(...args: any[]) => void} log
+ */
+
+/**
+ * @type {(options: LoggerOptions) => Logger}
+ */
+const createLogger = (options = {}) => {
```

Comment:
Just wondering why we are creating a brand new logger instead of using `@strapi/logger` with a custom config?
This would let user keep their own config (if they want to send their log using any other transport)

Reply:
I also think there's a whole load of inconsistencies in this package where there are a multitude of different ways we log stuff, I _think_ this is a _bit_ consistent with what DTS cli does.